tests/Unit/Collector/ProfileClientTest.php

```diff
@@ -34,7 +35,7 @@ class ProfileClientTest extends TestCase
     private $activeStack;
 
     /**
-     * @var HttpClient
+     * @var HttpClient|MockObject
```

Comment:
right.

just a comment, no need to change anything now: i have seen the syntax `HttpClient&MockObject` which is semantically correct and needed from some phpstan level on so phpstan is sure about the type of the variable. however, e.g. phpstorm seems not to know about it...

Reply:
Yea, it's right, PhpStorm don't suggest any method of MockObject so I should to add this,